cranelift/codegen/src/isa/x64/inst/mod.rs

```diff
@@ -886,19 +859,16 @@ impl MachInst for Inst {
         }
     }
 
-    fn gen_move(dst_reg: Writable<Reg>, src_reg: Reg, ty: Type) -> Inst {
+    fn gen_move(dst_reg: Writable<Reg>, src_reg: Reg, _ty: Type) -> Inst {
         let rc_dst = dst_reg.to_reg().get_class();
         let rc_src = src_reg.get_class();
         // If this isn't true, we have gone way off the rails.
         debug_assert!(rc_dst == rc_src);
         match rc_dst {
             RegClass::I64 => Inst::mov_r_r(true, src_reg, dst_reg),
-            RegClass::V128 => match ty {
-                F32 => Inst::xmm_r_r(SseOpcode::Movss, src_reg, dst_reg),
-                F64 => Inst::xmm_r_r(SseOpcode::Movsd, src_reg, dst_reg),
```

Comment:
Bad rebase here: I think the previous code can be reused effectively by replacing `xmm_r_r` with `xmm_mov_rm_r` and adapting to this function's API. Can you do this here please?